lib/i3ipc/connection.rb

```diff
@@ -36,6 +48,25 @@ def workspaces
       reply_for(1)
     end
 
+    def subscribe(event, block)
+      event_number = event_number(event)
+
+      # Send subscription request
+      @protocol.send(2, [event])
+
+      reply = Reply.parse(@protocol.receive 2)
+      raise WrongEvent.new(event) unless reply["success"]
```

Comment:
Ah, sorry about that! I did that initially and thought I committed it. Will push the right call now.